docs/utils/data.rst

```diff
@@ -0,0 +1,225 @@
+irregularity of the Earth's rotation) probably triggers a download of the
+IERS data::
+
+   >>> from astropy.time import Time
+   >>> Time.now().ut1  # doctest: +SKIP
+   Downloading https://maia.usno.navy.mil/ser7/finals2000A.all
+   |============================================| 3.2M/3.2M (100.00%)         1s
+   <Time object: scale='ut1' format='datetime' value=2019-09-22 08:39:03.812731>
+
+But running it a second time does not require any new download::
+
+   >>> Time.now().ut1  # doctest: +SKIP
+   <Time object: scale='ut1' format='datetime' value=2019-09-22 08:41:21.588836>
+
+Some data is also made available from the `Astropy data server`_ either
+for use within astropy or for your convenience. These are available more
+conveniently with the ``get_pkg_data_*`` functions::
+
+   >>> from astropy.utils.data import get_pkg_data_contents
+   >>> print(get_pkg_data_contents("coordinates/sites-un-ascii"))  # doctest: +SKIP
+   # these are all mappings from the name in sites.json (which is ASCII-only) to the "true" unicode names
+   TUBITAK->TÜBİTAK
+
+Usage From Outside Astropy
+==========================
+
+Users of Astropy can also make use of Astropy's caching and downloading
+mechanism. In its simplest form, this amounts to using
+`~astropy.utils.data.download_file` with the ``cache=True`` argument to obtain
+their data, from the cache if the data is there::
+
+   >>> from astropy.utils.iers import IERS_B_URL, IERS_B
+   >>> from astropy.utils.data import download_file
+   >>> IERS_B.open(download_file(IERS_B_URL, cache=True))["year","month","day"][-3:]  # doctest: +SKIP
+    <IERS_B length=3>
+    year month  day
+   int64 int64 int64
+   ----- ----- -----
+    2019     8     4
+    2019     8     5
+    2019     8     6
+
+If users want to update the cache to a newer version of the
+data (note that here the data was already up to date; users
+will have to decide for themselves when to obtain new versions),
+they can use the ``update_cache`` argument::
+
+   >>> IERS_B.open(download_file(IERS_B_URL,
+   ...                           cache=True,
+   ...                           update_cache=True)
+   ... )["year","month","day"][-3:]  # doctest: +SKIP
+   Downloading http://hpiers.obspm.fr/iers/eop/eopc04/eopc04_IAU2000.62-now
+   |=========================================| 3.2M/3.2M (100.00%)         0s
+   <IERS_B length=3>
+    year month  day
+   int64 int64 int64
+   ----- ----- -----
+    2019     8    18
+    2019     8    19
+    2019     8    20
+
+If they are concerned that the primary source of the data may be
+overloaded or unavailable, they can use the ``sources`` argument
+to provide a list of sources to attempt downloading from, in order.
+This need not include the original source. Regardless, the data
+will be stored in the cache under the original URL requested::
+
+   >>> f = download_file("ftp://ssd.jpl.nasa.gov/pub/eph/planets/bsp/de405.bsp",
+   ...     cache=False,
```

Comment:
To be consistent with the text that claims "the data will be stored in the cache"
```suggestion
   ...     cache=True,
```

Reply:
Good point. We're not relying on the doctest anyway.